src/components/Rockets.jsx

```diff
@@ -0,0 +1,65 @@
+import React, { useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { getRockets, bookingRocket, cancelBooking } from '../redux/rocketsSlice';
+import '../styles/Rockets.css';
+
+const Rockets = () => {
+  const dispatch = useDispatch();
+  const rockets = useSelector((state) => state.rockets.rockets);
+
+  useEffect(() => {
+    if (rockets.length === 0) {
+      dispatch(getRockets());
+    }
+  }, [dispatch, rockets.length]);
+
+  const handleBooking = (rocketId, reserved) => {
+    if (reserved) {
+      dispatch(cancelBooking(rocketId));
+    } else {
+      dispatch(bookingRocket(rocketId));
+    }
+  };
+
+  return (
+    <div className="rockets__container">
+      {rockets.map((rocket) => (
+        <div className="rockets__card" key={rocket.id}>
+          <img
+            className="rockets__card__img"
+            src={rocket.flickr_images?.[0]}
+            alt={rocket.rocket_name}
+          />
+          <div>
+            <h2 className="rockets__card__title">{rocket.rocket_name}</h2>
+            <p>
+              {rocket.isReserved && <span className="rockets__card__badge">Reserved</span>}
+              {rocket.description}
+            </p>
+            {!rocket.isReserved && (
+              <button
+                className="rockets__reserveBtn"
+                type="button"
+                onClick={() => handleBooking(rocket.id, rocket.isReserved)}
+              >
+                Book now!
+              </button>
```

Comment:
Please kindly use an `underline` for the `Rockets` navbar especially since it's the default page as this will help a visitor to know that it is the page displayed on first view and it is also what is visible in the design given 👍 

Design given :
![image](https://github.com/Dachrono/react-redux-project/assets/20537712/ca75e156-29e7-4519-8eda-43df23aa5e4f)

Reply:
Solved